lib/entice/logic/vitals.ex

```diff
@@ -21,44 +23,97 @@ defmodule Entice.Logic.Vitals do
   def unregister(entity_id),
   do: Entity.remove_behaviour(entity_id, Vitals.Behaviour)
```

Comment:
we need to remove both behaviours here. see group on how that can be done. removale is fail-safe, so it wont give you errors if you try to remove a non-existing behaviour. i.e. you can just remove both of them with no additional checks